localstack/services/route53resolver/provider.py

```diff
@@ -42,22 +46,22 @@
     Route53ResolverApi,
     TagList,
     UpdateFirewallDomainsResponse,
+    UpdateFirewallRuleGroupAssociationResponse,
     UpdateFirewallRuleResponse,
     ValidationException,
 )
 from localstack.services.route53resolver.models import Route53ResolverBackend
 from localstack.utils.aws import aws_stack
 from localstack.utils.collections import select_from_typed_dict
 from localstack.utils.patch import patch
-from localstack.utils.strings import get_random_hex
 
-
-def get_route53_resolver_firewall_rule_group_id():
-    return f"rslvr-frg-{get_random_hex(17)}"
-
-
-def get_route53_resolver_firewall_domain_list_id():
-    return f"rslvr-fdl-{get_random_hex(17)}"
+from .utils import (
```

Comment:
nit: could we use absolute imports here?

Reply:
I've added the absolute import